pkg/kubelet/kubelet.go

```diff
@@ -1334,6 +1354,11 @@ func (kl *Kubelet) initializeRuntimeDependentModules() {
 		// Fail kubelet and rely on the babysitter to retry starting kubelet.
 		glog.Fatalf("Failed to start ContainerManager %v", err)
 	}
+	// container log manager must start after container runtime is up to retrieve information from container runtime
+	// and inform container to reopen log file after log rotation.
+	if kl.containerLogManager != nil {
```

Comment:
Instead of having a nil containerLogManager, can we instead make a ContainerLogManagerStub for use with non-cri runtimes?  This way when we add other methods or calls to the ContainerLogManager interface, we dont need to do nil-checking everywhere

Reply:
Good idea.

Reply:
Done